backend/azure-functions/src/Services/DataAccess/vaccines-report.ts

```diff
@@ -0,0 +1,19 @@
+export type VaccinesReport = {
```

Comment:
Why twice? Why not use this from the libs interfaces library?

Reply:
Because the libs has an import that is not supported in naked typescript without nx.